Ziti.NET.Standard/msvc_build.bat

```diff
@@ -0,0 +1,102 @@
+@echo off
```

Comment:
this file is probably misnamed. doesn't seem like any 'msvc' stuff in here. perhaps just call it 'build.bat'?